src/plugin.js

```diff
@@ -84,9 +89,11 @@ class CimpressTranslationsWebpackPlugin {
     compiler.plugin("run", this.eventRun.bind(this));
     compiler.plugin("watch-run", this.eventRunDev.bind(this));
 
-    compiler.plugin("normal-module-factory", (nmf) => {
-      nmf.plugin("before-resolve", this.eventInterceptResolve.bind(this));
-    });
+    if (!this.skipDevelopmentUpdates) {
```

Comment:
I'm not sure I totally understand these lines.

Reply:
If you don't choose to skip the update on development,  the plugin will latch itself to a hook exposed by webpack's normal module factory (which itself is outputted by the compiler in another hook).

The hook is triggered (apparently asynchronously) for each element in the dependency tree as that tree is being constructed: at the point when the compiler has identified a new file to load after encountering a require/import statement calling for it, but before actually loading the file. The plugin holds off until the moment when one of the language files is attempted to be loaded before awaiting on the "download translations" promise and saving the translations.

If you choose to skip the update on development, there's no point in latching to the hook - because nothing will be saved anyway (what are going to await on?).